examples/tahoe-dev-toolkit.pkr.hcl

```diff
@@ -40,20 +40,20 @@ build {
     "macstadium-orka.image"
   ]
 
-provisioner "shell" {
-  execute_command = "echo 'admin' | sudo -S sh -c '{{ .Vars }} {{ .Path }}'"
-  inline = [
-    "echo 'Installing Homebrew'",
-    "echo 'Create homebrew directory with proper permissions'",
-    "mkdir -p /opt/homebrew",
-    "chown -R admin:admin /opt/homebrew",
-    "echo 'Install as the admin user, not root'",
-    "sudo -u admin NONINTERACTIVE=1 /bin/bash -c \"$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)\"",
-    "echo 'Remove temporary sudo access'",
-    "sudo rm -f /etc/sudoers.d/admin-temp",
-    "echo 'Homebrew installation completed and sudo access revoked'"
-  ]
-}
+  provisioner "shell" {
+    inline = [
+      "echo 'Setting up passwordless sudo for admin user'",
+      "echo 'admin' | sudo -S sh -c \"echo 'admin ALL=(ALL) NOPASSWD: ALL' > /etc/sudoers.d/admin-nopasswd\"",
```

Comment:
I know this is an example. But we are hard coding the username / password everywhere here. Where it should be paramaterized

This is a `nit` as again it is meant to be an example and we had it this way before, but it is the proper way

Reply:
no worries, updated this in a new commit